### ISSUE TITLE
Supported audio types

### ISSUE THREAD

**Issue by alexandergunnarson** [tlenclos/react-native-audio-streaming#15]
This issue is similar to https://github.com/tumtumtum/StreamingKit/issues/301 that I opened yesterday:

"I've gathered from issues on here and the examples you've included that these are the formats/codecs that are supported:

mp4 audio (m4a)
mp3
aac
wav
aiff (?)

Is this the case? Either way, could you do a quick update of the readme to get this info documented? It would be a huge help for me and probably other users of this excellent library."

I noticed that the iOS version of this library relies mainly on StreamingKit, so what I said applies to this library as well.

Thanks! Great work!

**Comment by tlenclos** [comment]
Good idea, this should be documented for both libraries and will need to be investigated.

**Comment by PetarDambovaliev** [comment]
.m4a doesn't work, play button loads for a bit and nothing else happens.
Any ideas?
